runelite-client/src/main/java/net/runelite/client/plugins/wintertodt/WintertodtOverlay.java

```diff
@@ -50,10 +50,13 @@ private WintertodtOverlay(WintertodtPlugin plugin)
 		getMenuEntries().add(new OverlayMenuEntry(RUNELITE_OVERLAY_CONFIG, OPTION_CONFIGURE, "Wintertodt overlay"));
 	}
 
+	@Inject
+	private WintertodtConfig config;
```

Comment:
Inject this in the constructor in the same way that `plugin` is injected in this class.